### PR TITLE
(MAINT) Update Dependencies

### PR DESCRIPTION
**Pull request by Dakta**
Make Beaker dependency permissive to support 4.0, and remove conflicting pry dependency. No need to specify it here, since we'll get some version via Beaker.

### CODE REVIEW

**Comment by Dakta** [comment]
I can't label this, but please consider this Do Not Merge until I can test it with someone's existing workflows.

**Comment by cthorn42** [comment]
@Dakta this looks good. I've tested this with a branch of pe_acceptance_tests that has a modified gem file that requires Beaker 4.0, a local beaker-http, a local Scooter (which needs the same modification that is made here).
If I use the master branch of beaker-http I get this error when running bundle install:
```
Bundler could not find compatible versions for gem "beaker":
  In Gemfile:
    beaker-http was resolved to 0.1.0, which depends on
      beaker (~> 3.0)

    beaker-pe was resolved to 2.0.0, which depends on
      beaker (~> 4.0)
```

If I then checkout a branch based  off of this PR, bundle will install, and the acceptance tests will pass.

**Review by cthorn42 — APPROVED** [review]

**Comment by cthorn42** [comment]
@tvpartytonight do you have merge access on this?

**Pull request merged**
